jaxtyping/_ipython_extension.py

```diff
@@ -17,7 +17,7 @@
 # IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-from ._import_hook import _JaxtypingTransformer
+from ._import_hook import _JaxtypingTransformer, _Typechecker
```

Comment:
Nit: I've realised that neither of these should have underscores -- the fact that we're importing them means they're clearly not private.

Reply:
They are still sort of module-wide private, but I agree that changing the names makes sense :)